src/style.css

```diff
@@ -1,3 +1,53 @@
 body {
   margin: 0;
+  font-family: 'Roboto', sans-serif;
+  background-color: lightgray;
+}
+
+header {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  padding: 5px 50px;
+  background-color: rgba(255, 255, 255, 1);
+}
+
+ul {
+  display: flex;
+  list-style: none;
+}
+
+li {
+  margin: 0 10px;
+  font-size: small;
+  font-weight: bold;
+}
+
+img {
+  margin-top: 5px;
```

Comment:
Avoid fixing container size. The 'img' tag has a fixed margin-top, which could lead to issues if the content above it changes size. Consider using a more flexible approach to spacing that doesn't rely on fixed sizes unless there's a specific requirement for it.